service/version_bundles.go

```diff
@@ -373,6 +373,11 @@ func NewVersionBundles() []versionbundle.Bundle {
 					Description: "Delete AWS Cloud Provider resources when deleting clusters.",
 					Kind:        versionbundle.KindAdded,
 				},
+				{
+					Component:   "aws-operator",
+					Description: "Get back Kubernetes Cluster ID tag helping cost analysis in AWS.",
```

Comment:
To me `Add KubernetesCluster AWS resource tag that was removed in 2.0.0.`  makes more sense.

I think the changelog should describe what is changing. The tag helps cost analysis but can be used for other things.

Reply:
ok, but then I leave the old tag name?

Reply:
Lets use the new cluster name and the changelog can be something like

`Add AWS resource tag with GiantSwarm Cluster ID for reporting.`